### PR TITLE
Update note on Win Server support

### PR DESCRIPTION
**Pull request by usha-mandya**
Update the note in Desktop docs to clarify that Windows Server isn't supported on Docker Desktop.

### CODE REVIEW

File: docker-for-windows/faqs.md [hunk 1]
```diff
@@ -117,9 +117,14 @@ Yes, you can run VirtualBox along with Docker Desktop if you have enabled the [
 
 ## Windows requirements
 
+### Can I run Docker Desktop on Windows Server?
+
+No, running Docker Desktop on Windows Server is not supported.
+
 ### How do I run Windows containers on Docker Desktop on Windows Server?
```
Comment (mat007) [inline docker-for-windows/faqs.md:124]: Do we want to change this as well?

Comment (usha-mandya) [reply docker-for-windows/faqs.md:124]: Could you provide more info? Do you mean updating the heading or combining both questions?

Comment (mat007) [reply docker-for-windows/faqs.md:124]: Oh sorry, I meant the heading looks wrong as it has Docker Desktop in it, maybe it should just be `How do I run Windows containers on Windows Server?`?

Comment (usha-mandya) [reply docker-for-windows/faqs.md:124]: Aah, yes. That makes sense.

Comment (usha-mandya) [reply docker-for-windows/faqs.md]: Updated

**Pull request merged**